gitgud/hooks/precommit.py

```diff
@@ -1,5 +1,4 @@
 import sys
 
-print('Command "git commit" is diabled. Use "git gud commit" instead.')
+print('Command "git commit" is disabled. Use "git gud commit" instead.')
 sys.exit(1)
```

Comment:
You can just remove the import statement and change `sys.exit(1)` to`exit(1)`. `exit` is a built-in

Reply:
done